.github/workflows/tests_reusable.yml

```diff
@@ -175,9 +170,37 @@ jobs:
           date
           ACCEPTANCE_TESTS='(TestAccResourceTaikunOrganization|TestAccDataSourceTaikunOrganization|TestAccDataSourceTaikunShowback|TestAccDataSourceTaikunAccessProfile|TestAccResourceTaikunAccessProfile|TestAccDataSourceTaikunAlertingProfile|TestAccResourceTaikunProjectModifyAlertingProfile|TestAccResourceTaikunKubernetesProfile|TestAccResourceTaikunBilling|TestAccResourceTaikunProjectModifyImages|TestAccResourceTaikunProjectD|TestAccResourceTaikunProjectK|TestAccResourceTaikunUser|TestAccResourceTaikunShowback|TestAccResourceTaikunProjectModifyFlavors|TestAccResourceTaikunProjectU|TestAccResourceTaikunProject$$|TestAccDataSourceTaikunCloudCredentialOpenStack|TestAccDataSourceTaikunCloudCredentialsOpenStack|TestAccDataSourceTaikunCloudCredentialAzure|TestAccDataSourceTaikunCloudCredentialsAzure|TestAccDataSourceTaikunCloudCredentialAWS|TestAccDataSourceTaikunCloudCredentialsAWS|TestAccDataSourceTaikunKubernetesProfile|TestAccDataSourceTaikunBillingRule|TestAccDataSourceTaikunBillingCredential|TestAccDataSourceTaikunBackupCredential)' ./scripts/rerun_failed_tests.sh
 
+  test1c:
+    name: 1c) Dry Test GCP
+    needs: [build,golangci]
+    runs-on: self-hosted
+    timeout-minutes: 30
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.BRANCH }}
+      - name: Get dependencies
+        run: |
+          go mod download
+      - name: Create GCP config file
+        env:
+          GCP_CONFIG_FILE: ${{ secrets.GCP_CONFIG_FILE }}
+        run: |
+          echo "$GCP_CONFIG_FILE" > ./taikun/gcp.json
+      - name: TF acceptance tests
+        run: |
+          date
+          ACCEPTANCE_TESTS='(TestAccResourceTaikunCloudCredentialGCP$$|TestAccDataSourceTaikunCloudCredentialsGCP$$|TestAccDataSourceTaikunImagesGCP$$|TestAccDataSourceTaikunCloudCredentialGCP$$)'
+          go test . ./taikun -v -run ${ACCEPTANCE_TESTS} -timeout 30m
+
   test2a:
     name: 2a) Openstack, creating instances
```

Comment:
It would be nice to have specific and descriptive test names

Reply:
Good idea! Improved in [10f38dd](https://github.com/itera-io/terraform-provider-taikun/pull/295/commits/10f38dd56eb1272edb6f8fae3d208e7d5a4a7eaf)